### DISCUSSION TITLE
Streamed Text Input to Whisper TTS

### DISCUSSION THREAD

**Discussion by saduri2004** [openai/whisper#1963] [Q&A]
I know that there is a way to stream the audio file output, as seen in the Whisper Docs, but I was wondering if anyone has found a way (or if it is even possible to engineer a way) to stream the input. I am taking the output of the completions API call (where streamed output is set to TRUE), so as the stream comes in from completions endpoint, I want to speak it out at the same time. Is this feasible?

**Comment by makaveli10** [comment]
@saduri2004 give [WhisperLive](https://github.com/collabora/WhisperLive) a try it supports streaming to the server running Whisper model.

**Reply by jayavanth** [reply]
@makaveli10 curious how it works. Is it chunking the input?
